docs/deploy/install.md

```diff
@@ -0,0 +1,63 @@
+# Installation
+
+**Requires Kubernetes 1.22 or newer**. Learn how to [install the Kubernetes tools](https://kubernetes.io/docs/tasks/tools/).
+
+You can deploy Kueue in your cluster with the following command:
+
+```sh
+IMAGE_REGISTRY=registry.example.com/my-user make image-build image-push deploy
+```
+
+The controller will run in the `kueue-system` namespace.
+
+```sh
+$ kubectl get all -n kueue-system
+NAME                                            READY   STATUS    RESTARTS   AGE
+pod/kueue-controller-manager-57d966978b-h88sg   2/2     Running   0          36s
+
+NAME                                               TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)    AGE
+service/kueue-controller-manager-metrics-service   ClusterIP   10.104.161.54   <none>        8443/TCP   36s
+
+NAME                                       READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/kueue-controller-manager   1/1     1            1           36s
+
+NAME                                                  DESIRED   CURRENT   READY   AGE
+replicaset.apps/kueue-controller-manager-57d966978b   1         1         1       36s
+```
+
+## Uninstall
+
+Once you are done, you can simply run
```

Comment:
```suggestion
To uninstall Kueue, run the following command:
```

Do we really need to show the output?

Reply:
is a nice to have in documentation. not needed, but sometimes I think it helps the users follow through